main.py

```diff
@@ -1 +1,35 @@
-rint("hello")
+import mysql.connector
+from mysql.connector import Error
+from fastapi import FastAPI
+from pydantic import BaseModel
+import uvicorn
+
+app = FastAPI()
+
+db_config = {
+    "host": "35.239.158.202",
+    "user": "root",
+    "password": "7$x[;*X-0qpT[J2n",
+    "database": "test",
+}
```

Comment:
These things are hardcoded and that is not acceptable. You have to read these values from environment variables. Use `os.environ['HOST']` and here is the example how you should run the script (ofc, you will add other variables as well):
`HOST='35.239.158.202' USER='root' python main.py`